automation/roles/eda-deploy-server/templates/eda-server-pvc.yaml.j2

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
```

Comment:
If my comment above is true, then we can remove this file and the reference to it.